src/Core/Renderer/DefaultPerceivedRoomRenderer.cs

```diff
@@ -31,22 +31,42 @@ public override string Render(Thing viewer, Thing room)
             //   var exits = senses.PerceiveExits();        and also render closable exits like doors nicely; "(closed)"?
             //   var entities = senses.PerceiveEntities();  and also render players nicely; "(AFK)" etc?
             //   var items = senses.PerceiveItems();        and also track tick or build sense-specific strings (like hearing only while blind...)
+
             bool hasNoticedSomething = false;
+
+            // Handle exits differently from other Thing types
+            var exits = senses.PerceiveExits();        // TODO: Aso render closable exits like doors nicely; "(closed)"?
+            if (exits.Count > 0)
+            {
+                sb.Append($"  routes: ");
+                foreach (var exit in exits)
+                {
+                    sb.Append($"<%magenta%>{exit}<%n%>, ");
+                    hasNoticedSomething = true;
+                }
+                sb.Length--;
+                sb.Length--;
+                sb.AppendLine();
```

Comment:
I think we want to avoid AppendLine now, as I think that'll add the server's OS `Environment.NewLine` but might not be \r\n - thus technically not adhering strictly to the Telnet protocol, as servers are expected to send CR LF regardless of which environment it is running on.  (It's possible that under the hood somewhere we're converting all line ending combos to CR LF but I'm not sure offhand where if we do.)  So I recently created the `AnsiSequences.NewLine` to serve this purpose.  :)

Finally, this might be a good opportunity to learn some cool tricks with C#, if you want.  The approach you have here is acceptable with the other minor adjustments noted above.  But if you like, could you test if something like this would work as well, with `using System.Linq;`
```
var coloredExits = from exit in senses.PerceiveExits() select $"<%magenta%>{exit}<%n%>";
if (coloredExits.Any())
{
    sb.Append($"  Routes: {string.Join(", ", coloredExits)}{AnsiSequences.NewLine}");
}
```
We could even do away with `hasNoticedSomething` entirely by applying the same approach to the non-exit Things as well, by using the LINQ `where` keyword to exclude things that have ExitBehavior via the LINQ query.

Reply:
A StringBuilder extension method would be neat.  If you want to t tryhat, go for it.  We have a wide Extensions.cs in Utilities that... looks like both the things in there are unused now actually and should probably get deleted.  Anyway, we could call it `AppendAnsiLine` so that IntelliSense will help remind the dev about it as they start typing `sb.Append...`

Reply:
I've implemented the AppendAnsiLine extension method and using it to produce output like the following : 

<img width="582" alt="issue-67-pr-output" src="https://user-images.githubusercontent.com/55847560/104529540-e44ade80-5601-11eb-991f-532daa6c9abc.png">

Hopefully will have time to investigate the LINQ approach you mentioned but am back to work so it might take a bit depending on what I find when I start looking into it.